python/lsst/obs/decam/ingestCalibs.py

```diff
@@ -105,3 +82,26 @@ def getExtensionName(md):
         @param md (PropertySet) FITS header metadata
         """
         return md.get('EXTNAME')
+
+    def getDestination(self, butler, info, filename):
```

Comment:
Would it be possible to use the base method and then add necessary overrides? 

Also I worry what this means to ingest the LSST-generated calibration data for DECam, the scencario @mfisherlevine has been using. Somebody would want to test that scenario before merging.